packages/experiments/src/components/fluent/FluentStyles.ts

```diff
@@ -139,14 +139,15 @@ const LabelStyles = {
 };
 const ToggleStyles = {
   pill: {
-    width: '2em',
-    height: '1em',
-    borderRadius: '.5em'
+    width: '40px',
+    height: '20px',
+    borderRadius: '10px',
+    padding: '0 4px'
```

Comment:
Why the side padding?

Reply:
Because it's in ems in the pre-fluent component styles, so I added the px style here to uniformly update to px from em.

Reply:
Awesome!